src/tbbmalloc/large_objects.h

```diff
@@ -81,18 +81,25 @@ struct HugeBinStructureProps {
 
     static size_t alignToBin(size_t size) {
         MALLOC_ASSERT(size >= StepFactor, "Size must not be less than the StepFactor");
-        size_t minorStepExp = BitScanRev(size) - StepFactorExp;
+
+        int sizeExp = (int)BitScanRev(size);
+        MALLOC_ASSERT(sizeExp >= 0, "Size must be greater than 0");
```

Comment:
Can we re-use your wording like "BitScanRev cannot return -1 as size >=stepfactor >0"?

Reply:
Done